lib/Adapter/WorseReflection/Refactor/WorseGenerateAccessor.php

```diff
@@ -112,4 +105,41 @@ private function sourceFromSymbolInformation(SourceCode $sourceCode, SymbolConte
             $worseSourceCode->path()
         );
     }
+
+    /**
+     * Finds the class containing the property.
+     *
+     * @param string $source The source code of the file.
+     * @param int $offset The position where the cursor was, used to identify the correct
+     * class if there is more than one in the `$source`.
+     *
+     * @return ReflectionClass
+     *
+     * @throws InvalidArgumentException If there is no class in the code.
+     * @throws RuntimeException If it's impossible to determine which class to use.
+     */
+    private function class(string $source, int $offset): ReflectionClass
```

Comment:
We don't document methods _generally_ if the meaning can be reasonably inferred. We have the type hints, and the prose documentation above for each parameter would have logically repeated for every single method, and this is not a public API. `$offset` is always a byte offset in this library (it should be a value object, but that's another topic).

The `@throws` have more value, but for the sake of maintainence, and that this is not a public API, I'd also be happy to drop them.

`@return` is completely redundant.

Reply:
I agree, the main goal was to point out that the offset is used to identify the class only in a specific case.
But I'm totally fine withe the new name.